src/main/java/analyzer/AnalyzerRoot.java

```diff
@@ -49,6 +50,7 @@ private static List<Analyzer> createAnalyzers(String slug) {
             case "lasagna" -> analyzers.add(new LasagnaAnalyzer());
             case "leap" -> analyzers.add(new LeapAnalyzer());
             case "two-fer" -> analyzers.add(new TwoferAnalyzer());
+            case "need-for-speed" -> analyzers.add(new NeedForSpeedAnalyzer());
```

Comment:
Can you make sure the switch cases are ordered alphabetically?

Reply:
Done!